RileyLinkKit/PumpOpsSynchronous.swift

```diff
@@ -464,8 +464,17 @@ class PumpOpsSynchronous {
                     throw error
                 }
             }
-            
-            NSLog("Fetched page %d: %@", pageNum, pageData as NSData)
+            if true {
+                let logString = NSString(format: "Fetched page %d: %@", pageNum, pageData as NSData ) 
+                let sz = logString.length
```

Comment:
It should always be 1024 bytes.   4 lines of 256 bytes would be great (512 characters/line + some prefix to separate it from the rest of the log)
Something like `page 1 - bytes (000-255)` would be a nice prefix